dbcrossbar/fixtures/tricky_column_names.csv

```diff
@@ -0,0 +1,2 @@
+"person__Delivery Zone 4.14","person__$Region - 8/1","person__Do you have any allergies?",name,Name
+a,b,c,d,e
```

Comment:
@emk Would it be helpful to add columns `person__I've been` and `person__Subscription #` here? Found them in Stitch

Reply:
We certainly could!

But since we treat all non-identifier characters the same, it wouldn't actually prove anything new. But I'd be happy to take a patch if you're feeling distrustful. :-)

Reply:
@emk Gotcha! Since this won't prove anything new, I guess the only benefit would be that it will give the reader a better idea of the possible non-identifier characters Stitch can throw at us. Up to you